.github/workflows/backend.yml

```diff
@@ -154,8 +154,6 @@ jobs:
         uses: ./.github/actions/setup-python
         with:
           python-version: ${{ matrix.python-version }}
-          # Note this uses a different cache key than other backend-based workflows because this workflows dependencies are different
-          cache-files-hash: ${{ hashFiles('requirements-pre-commit.txt') }}
```

Comment:
this'll need to be back to `requirements-dev.txt` I think

Reply:
```
  cache-files-hash:
    description: 'A single hash for a set of files. Used for caching.'
    required: false
    default: ${{ hashFiles('requirements-*.txt') }}
```

Reply:
Oh wait, this is backend lint.